### PR TITLE
CAMEL-21269: camel-langchain4j-tokenizer fixes for the YAML DSL

### PR DESCRIPTION
**Pull request by orpiske**
This is a more more comprehensive fix for the `tokenizer` support for the YAML DSL.  There were quite a few problems in the model files that prevented it from being used with YAML (thus affecting any sub-project or project relying on this). 

Although it's relatively big due to the regen of a few files, all the changes are backwards compatible and anyone using it with the Java DSL should not have to change any code.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json [hunk 1]
```diff
@@ -7267,6 +7267,100 @@
...
+            "type" : "boolean",
+            "title" : "Disabled",
+            "description" : "Whether to disable this EIP from the route during build time. Once an EIP has been disabled then it cannot be enabled later at runtime."
+          },
+          "id" : {
+            "type" : "string",
+            "title" : "Id",
+            "description" : "Sets the id of this node"
+          },
+          "langChain4jCharacterTokenizer" : { },
```
Comment (davsclaus) [inline dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json:7291]: This looks a bit suspesious that these are here and empty

Comment (orpiske) [reply dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json:7291]: Hm ... interesting (maybe that's why we are not seeing them on the dropdown, @lordrip). 

Thanks @davsclaus, let me investigate this.

Comment (orpiske) [reply dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json:7291]: @davsclaus upon closer inspection this seems fine. We have the same thing for the Marshal definition (which was what I based myself for adjusting this): https://github.com/apache/camel/blob/db7a2bb33eb9d2501134ca9b393d7281d7a1c56f/dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json#L3251-L3257

Comment (davsclaus) [reply dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json:7291]: okay thanks

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
You should also ping @mgubaidullin as it can affect Camel Karavan

**Comment by orpiske** [comment]
> You should also ping @mgubaidullin as it can affect Camel Karavan

Absolutely. I couldn't reproduce w/ the existing version of Karavan, but even if it got everything correctly, the generated YAML would still be wrong.

It's on my TODO list for next week to figure out if/how Karavan is affected so I can pinpoint exactly where/how we need to fix any code relying on the incorrect model.

**Comment by orpiske** [comment]
I've just confirmed that the YAML itself is functional. Now I'll work with the users / tooling creators to make sure they can leverage the new EIP. 

Let's merge this to make it easily available for others.

**Pull request merged**